database/database.js

```diff
@@ -0,0 +1,48 @@
+const mongoose = require('mongoose');
+
+mongoose.connect('mongodb://localhost/metadata', {useNewUrlParser: true, useUnifiedTopology: true})
+    .catch(error => handleError(error));
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log('We\'re connected!');
+});
+
+//create our metadata Schema
+const metaSchema = new mongoose.Schema({
```

Comment:
I actually ran into this problem, but I notice that you are building out the same schema twice. Once in your database file and then again in your seed script file. I would recommend building it only in your database.js and then exporting it to your seed script rather than writing it again in there.